kibana/examples/security/Makefile

```diff
@@ -3,8 +3,16 @@ default: install
 
 RELEASE := helm-kibana-security
 
-install:
+install: secrets
 	helm upgrade --wait --timeout=600 --install --values ./security.yml $(RELEASE) ../../ ; \
 	
 purge:
 	helm del --purge $(RELEASE)
+
+secrets:
+	kubectl delete secret kibana-certificates || true
+	vault read -field=kibana.crt secret/devops-ci/helm-charts/kibana/security/certificates | base64 --decode > kibana.crt
```

Comment:
Possibly outside the scope of this PR, but we could remove the dependency on vault here by generating self-signed certs with `openssl` -- thoughts?

Reply:
This is actually true for all of the testing secrets except for the license. Although for users without a license hitting the trial API would also be acceptable for testing. I'm explicitly using a real license though because I know that some rules are more relaxed when using a trial (such as no node to node encryption being required). 

I'm going to make an issue to make sure this gets done. End results is to make sure that all users and contributors can deploy the examples with minimal effort.